File: tarakci_emre.php
<?php
   $i = 0;
    while ($i < 3) {
        echo "-- Loop when i = $i\n";
        $i++;
        while (true) {
            while (true) {
                echo "This is the inner loop\n";
                continue 3;
            }
            echo "This is the loop in the middle\n";
        }
    }
    
    echo "\n";
    
    for ($j = 0; $j < 3; $j++) {
        echo "-- Loop when j = $j\n";
        while(true) {
            break;
            echo "The inner loop\n";    
        }
    }
    
    echo "\n";
    // What happens when break is used with number
    for ($j = 0; $j < 3; $j++) {
        echo "-- Loop when j = $j\n";
        while(true) {
            break 2;
        }
    }
?>

File: tarakci_emre.rb
for i  in  0..5
    puts "Loop at i = #{i}:"
    if  i % 2 == 0 then
        next
    end
    j = 1
    while true
        puts "j: #{j}"
        j += 1
        break if j > 2
    end
end

